onnx/defs/schema.h

```diff
@@ -258,6 +272,27 @@ class OpSchema {
 };
 
 /**
+ * Internal class used in schema declaration
+ */
+class OpSchemaHolder {
+ public:
+  OpSchemaHolder(OpSchema& schema) : schema_(&schema) {
```

Comment:
nit: there is no need to store `schema_`

Reply:
I kept it just in case somebody would want to access schemas through global variables (probably not)